lib/less/math-constants.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+    ALWAYS: 0,
+    PARENS_DIVISION: 1,
+    PARENS_ALL: 2,
+    STRICT_LEGACY: 3
```

Comment:
Speaking of style guides, you have any strong opinion on comma dangle?

Reply:
Comma dangles shouldn't go in, probably, for compatibility reasons.